packages/github/webhook.js

```diff
@@ -0,0 +1,64 @@
+var request = require('request');
```

Comment:
Try to have a better commit message. I personally like the openstack style.
One line as a brief introduction.
Then more detailed descriptions in one or multiple paragraphs.
In the end put the bug number.